src/Workspaces/Core/Portable/FindSymbols/Declarations/DeclarationFinder.cs

```diff
@@ -107,9 +107,8 @@ internal static partial class DeclarationFinder
                 if (reference != null)
                 {
                     var info = await SymbolTreeInfo.GetInfoForMetadataReferenceAsync(
-                        project.Solution, reference, loadOnly: false, cancellationToken).ConfigureAwait(false);
+                        project.Solution, reference, cancellationToken).ConfigureAwait(false);
 
-                    // We must have an index since we passed in loadOnly: false here.
                     Contract.ThrowIfNull(info);
```

Comment:
probably not necessary (we have NRT on these apis).  but it doesn't hurt to still have this check.